<ui version="4.0" >
 <class>PopupPublicBase</class>
 <widget class="QWidget" name="PopupPublicBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>562</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="searchLabel" >
       <property name="text" >
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KTreeWidgetSearchLine" name="listSearch" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="keyList" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Email</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>ID</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGroup" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="untrustedCheck" >
        <property name="whatsThis" >
         <string>&lt;b>Allow encryption with untrusted keys&lt;/b>: when you import a public key, it is usually marked as untrusted and you cannot use it unless you sign it in order to make it 'trusted'. Checking this box enables you to use any key, even if it has not be signed.</string>
        </property>
        <property name="text" >
         <string>Allow encryption with untrusted keys</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTreeWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
